nym-vpn-core/crates/nym-vpnd/src/service/config/v4.rs

```diff
@@ -68,6 +68,9 @@ impl TryFrom<VpnServiceConfig> for nym_vpn_lib_types::VpnServiceConfig {
             enable_custom_dns: value.enable_custom_dns,
             custom_dns,
             network_stats: nym_vpn_lib_types::NetworkStatisticsConfig::from(value.network_stats),
+            average_packet_delay: None,
+            message_sending_average_delay: None,
+            poisson_parameter: None,
```

Comment:
At some point you need to add your options to a version of `VpnServiceConfig`. 
Current latest is v4, but it's getting code frozen for the release on Thursday Dec 11th so you will probably need a v5